graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/GraphQLMetricHandler.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018-2019 Expedia, Inc.
+ * Copyright (C) 2018-2020 Expedia, Inc.
```

Comment:
Can we do a project wide find/replace, get these all updated in one go?

Reply:
Only the files that have been edited in 2020 get updated